BuildingThemes/Configuration.cs

```diff
@@ -72,15 +73,18 @@ public class Building
             [XmlIgnoreAttribute]
             public bool isBuiltIn = false;
 
+            [XmlAttribute(" include"), DefaultValue(true)]
```

Comment:
the space character in this string causes that the config is not loaded correctly.